packages/fast-components-react-msft/src/text-action/text-action.props.ts

```diff
@@ -46,6 +46,20 @@ export interface TextActionHandledProps
      * The trailing glyph
      */
     afterGlyph?: (classname?: string) => React.ReactNode;
+
+    /**
+     * Callback for when text action input is focused
+     */
+    onFocus?: (
```

Comment:
is there a reason these are mouse/keyboard events and not focus events?

Reply:
Nope, that was an error - I fixed it